### PR TITLE
merge imports

### PR DESCRIPTION
**Pull request by meetar**

### CODE REVIEW

File: themes/terrain-texture.yaml
Comment (nvkelso) [inline themes/terrain-texture.yaml:178]: please always use the `-` or `[]` instead of some of each.

Comment (nvkelso) [reply themes/terrain-texture.yaml:178]: (You're mostly using `-`, but then @sensescape is using `[]`, please standardize that in this PR, probably to `-`.)

**Review by nvkelso — APPROVED** [review]
Other than my always `-` comment, LGTM.

Let's also get a review from @sensescape before merging.

File: themes/terrain-texture.yaml
Comment (meetar) [reply themes/terrain-texture.yaml:178]: addressed with 68f2129a590b3290752febfff9fc33cb7d056fb3

Comment (meetar) [reply themes/terrain-texture.yaml:178]: Oops, just saw this – the `-` came from the yaml-merging tool I used to squish all the imports together, I tend toward `[]` for brevity but legibility is good too, I'll switch them back over.

**Review by sensescape — COMMENTED** [review]
LGTM

**Pull request merged**
